apps/demo/app/custom-ui/[...puckPath]/client.tsx

```diff
@@ -309,6 +314,10 @@ export function Client({ path, isEdit }: { path: string; isEdit: boolean }) {
         data={data}
         iframe={{ enabled: false }}
         headerPath={path}
+        permissions={{
+          debug: true,
+          test: true,
+        }}
```

Comment:
Might be nice to showcase this working fully by disabling the `debug` permission on a specific component

Reply:
I didn't only on the basis that it would be in the config that is shared for custom-ui and demo

Reply:
Created an override for this